lib/firmware/fsm_msg_tendermint.h

```diff
@@ -154,7 +154,9 @@ void fsm_msgTendermintMsgAck(const TendermintMsgAck *msg) {
     case OutputAddressType_TRANSFER:
     default: {
       char amount_str[32];
-      bn_format_uint64(msg->send.amount, NULL, msg->denom, 6, 0, false, amount_str,
+      char suffix[sizeof(coin->coin_shortcut) + 1];
+      snprintf(suffix, sizeof(suffix), " %s", coin->coin_shortcut);
+      bn_format_uint64(msg->send.amount, NULL, suffix, coin->decimals, 0, false, amount_str,
```

Comment:
on <=7.1.4 you'd get a confirm message like `0.01 ATOM`, but after #277 you'd get `0.01uatom`.

Reply:
Isn't uatom correct in this case?

Reply:
not if you divide by 10^6 first!

Reply:
cool